tests/testthat/test-predict-boots.R

```diff
@@ -1,19 +1,17 @@
-# read in data to use in tests
```

Comment:
I moves these to a "helper.R" file. Since it starts wit "helper" it gets executed prior to each test file.